Shared/Scripts/bxl.ps1

```diff
@@ -560,7 +560,7 @@ if (!$skipFilter){
 
     if ($Minimal) {
         # filtering by core deployment.
-        $AdditionalBuildXLArguments +=  "/f:(output='$($useDeployment.buildDir)\*'or(output='out\bin\$DeployConfig\Sdk\*')or($CacheOutputFilter))and~($CacheLongRunningFilter)ortag='protobufgenerator'oroutput='cg\*'"

```

Comment:
>'oroutput='cg\*'" [](start = 198, length = 17)

I might be missing something, why can we remove this filter?

Reply:
because there are no build outputs under the `cg` folder any longer